modules/dreamview/frontend/src/utils/models.js

```diff
@@ -51,3 +51,7 @@ export function loadObject(materialFile, objectFile, scale, callback) {
 export function loadTexture(textureFile, onLoadCallback, onErrorCallback) {
     textureLoader.load(textureFile, onLoadCallback, undefined, onErrorCallback);
 }
+
+export function loadMaterial(materialFile, onLoadCallback, onErrorCallback) {
+    mtlLoader.load(materialFile, onLoadCallback, undefined, onErrorCallback);
```

Comment:
Looks like we never pass in onErrorCallback? Is just `mtlLoader.load(materialFile, onLoadCallback);` okay?